package-lock.json

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
Adding this file to the repo is unrelated to a port to N-API. If adding a `package-lock.json` is desired, it should be done in a separate PR.

Reply:
Thus, IMO this file should be removed from this PR.

Reply:
Done 95fea6ad9493fed11dd72c709d73273c07d0fd29